moveit_ros/perception/lazy_free_space_updater/src/lazy_free_space_updater.cpp

```diff
@@ -121,6 +121,21 @@ void LazyFreeSpaceUpdater::processThread()
     if (!running_)
       break;
 
+    if (!process_occupied_cells_set_ || !process_model_cells_set_)
+    {
+      if (process_occupied_cells_set_)
```

Comment:
This line is not necessary because ```delete (NULL)``` is valid.

Reply:
Fix it in commit 573a03a.

Reply:
OK.